dom0/sd-default-config.sls

```diff
@@ -17,7 +17,7 @@
 {% if d.environment == "dev" %}
   # use apt-test and nightlies
   {% set sdvars = sdvars_defaults["test"] %}
-  {% set _ = sdvars.update({"component": "nightlies"}) %}
+  {% set _ = sdvars.update({"component": "main nightlies"}) %}
```

Comment:
This breaks: <https://github.com/freedomofpress/securedrop-workstation/blob/941f051f2e7fb627935dc661d01e7504b3b7c884/tests/test_vms_platform.py#L36-L37>

Reply:
the kernel string test failure will be addressed with next 6.6 kernels, @legoktm already fixed that in a PR pending to `kernel-builder`.